<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting</class>
 <widget class="QDialog" name="setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统设置界面</string>
  </property>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_name">
    <property name="text">
     <string>       快递箱名称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_name"/>
  </widget>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_middle">
    <property name="text">
     <string>中号箱子</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_middle"/>
  </widget>
  <widget class="QSplitter" name="splitter_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_small">
    <property name="text">
     <string>小号箱子</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_small"/>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_big">
    <property name="text">
     <string>大号箱子</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_big"/>
  </widget>
  <widget class="QPushButton" name="pb_create">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>146</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>完成设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_return">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>211</y>
     <width>146</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>451</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QPushButton" name="pb8">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pb9">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pb3">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pbDelete">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pb0">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pb2">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pb6">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pb7">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pb5">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pb4">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pb1">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pbpoint">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_ip">
    <property name="text">
     <string>        服务器IP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_ip"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
